### PR TITLE
Update application.py    project1-task2

### PR DESCRIPTION
**Pull request by saragadamraviteja**

### CODE REVIEW

**Comment by manasa133** [comment]
- check password is strong or not at html or  in server. 

- after successfull registration, render the template with a message saying successfull or not

**Comment by saragadamraviteja** [comment]
> * check password is strong or not at html or  in server.
> * after successfull registration, render the template with a message saying successfull or not

Implemented Madam...

**Comment by manasa133** [comment]
Generally we will save username and password in DB for login purpose, contanct numbe doesn't seen  correct

File: project1/application.py [hunk 1]
```diff
@@ -30,12 +35,20 @@ def index():
...
+        print(name)
         pswd = request.form.get("Password")
         print(pswd)
-        return render_template("login.html",name=name)
-
-    return render_template("register.html")
+        
+        contact = request.form.get("ContactNumber")
+        print(contact)
+        regist = data(username=name,ContactNumber=contact)
```
Comment (manasa133) [inline project1/application.py]: contact number doesnt seem right, better to save uname and passwrd

Comment (saragadamraviteja) [reply project1/application.py]: okay madam i will change..

**Pull request closed without merging**